src/components/Dissolution/CompleteAffidavit.vue

```diff
@@ -123,6 +123,11 @@
             <span class="ml-2">Must be set to fit onto 8.5" x 11" letter-size paper
             </span>
           </li>
+          <li class="mt-1">
+            <v-icon>mdi-circle-small</v-icon>
+            <span class="ml-2">Allow a minimum 1.5" margin at the top of the first page
+              (for BC Registries certified stamp)</span>
+          </li>
```

Comment:
This li tag and the same ones below are for the ticket [11316](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/11316): Create UI - CP Dissolution - Update text.